oxford-studies-in-ancient-philosophy.csl

```diff
@@ -126,9 +114,10 @@
     <group delimiter=". ">
       <names variable="author">
         <name name-as-sort-order="first" and="text" initialize-with="." sort-separator=", " delimiter=", " delimiter-precedes-last="always"/>
+       <label form="short" prefix=", "/>
```

Comment:
@adam3smith, tsk. Clearly didn't use http://formatter.citationstyles.org/ to reindent :P.